modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -105,7 +105,8 @@ def transform_disability_values!(disability)
 
       def check_for_pact_special_issue(disability)
         related_to_toxic_exposure = disability[:isRelatedToToxicExposure]
-        if related_to_toxic_exposure
+        action_not_increase = disability[:disabilityActionType] != 'INCREASE'
+        if related_to_toxic_exposure && action_not_increase
```

Comment:
@stiehlrod  Wonder if this check might be better up one level.  Not sure this method really needs to know about this?
Something like
```
        check_for_pact_special_issue(disability) if disability[:disabilityActionType] != 'INCREASE'
```
on line 101?

thoughts?

Reply:
@rockwellwindsor-va Good thought. thanks